recipes/poco/all/conanfile.py

```diff
@@ -1,85 +1,81 @@
-               "fPIC": [True, False],
-               "enable_xml": [True, False],
-               "enable_json": [True, False],
-               "enable_mongodb": [True, False],
-               "enable_pdf": [True, False],
-               "enable_util": [True, False],
-               "enable_net": [True, False],
-               "enable_netssl": [True, False],
-               "enable_netssl_win": [True, False],
-               "enable_crypto": [True, False],
-               "enable_data": [True, False],
-               "enable_data_sqlite": [True, False],
-               "enable_data_mysql": [True, False],
-               "enable_data_odbc": [True, False],
-               "enable_data_postgresql": [True, False],
-               "enable_encodings": [True, False],
-               "enable_jwt": [True, False],
-               "enable_sevenzip": [True, False],
-               "enable_zip": [True, False],
-               "enable_apacheconnector": [True, False],
-               "enable_cppparser": [True, False],
-               "enable_pocodoc": [True, False],
-               "enable_pagecompiler": [True, False],
-               "enable_pagecompiler_file2page": [True, False],
-               "enable_redis": [True, False],
-               "force_openssl": [True, False],
-               "enable_tests": [True, False],
-               "poco_unbundled": [True, False],
-               "cxx_14": [True, False]
-              }
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+    }
     default_options = {
-                "shared": False,
-                "fPIC": True,
-                "enable_xml": True,
-                "enable_json": True,
-                "enable_mongodb": True,
-                "enable_pdf": False,
-                "enable_util": True,
-                "enable_net": True,
-                "enable_netssl": True,
-                "enable_netssl_win": True,
-                "enable_crypto": True,
-                "enable_data": True,
-                "enable_data_sqlite": True,
-                "enable_data_mysql": False,
-                "enable_data_odbc": False,
-                "enable_data_postgresql": False,
-                "enable_encodings": True,
-                "enable_jwt": True,
-                "enable_sevenzip": False,
-                "enable_zip": True,
-                "enable_apacheconnector": False,
-                "enable_cppparser": False,
-                "enable_pocodoc": False,
-                "enable_pagecompiler": False,
-                "enable_pagecompiler_file2page": False,
-                "enable_redis": True,
-                "force_openssl": True,
-                "enable_tests": False,
-                "poco_unbundled": False,
-                "cxx_14": False
-            }
+        "shared": False,
+        "fPIC": True,
+    }
+
+    _PocoComponent = namedtuple("_PocoComponent", ("option", "default_option", "dependencies", "is_lib"))
```

Comment:
maybe in the future we can move it to a yaml file, IF Poco changes a lot, like Boost and new libs every new release.